maintainers/maintainer-list.nix

```diff
@@ -7089,6 +7089,12 @@
     githubId = 13000278;
     name = "Maksim Bronsky";
   };
+  PlushBeaver = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add PlushBeaver``.